### PR TITLE
kinder: add support for include workflow

### PR DESCRIPTION
**Pull request by fabriziopandini**
This PR adds support for including workflow

This enables the reuse of the same list of task in several workflows (as an alternative to make workflows configurable via env variables)

/assign @neolit123

/cc @kubernetes/sig-cluster-lifecycle-pr-reviews

### CODE REVIEW

File: kinder/pkg/test/workflow/workflow.go [hunk 1]
```diff
@@ -137,11 +148,77 @@ func NewWorkflow(file string) (*Workflow, error) {
...
+				continue
+			}
+			log.Warnf("env var %s in workflow file %s is shadowed by env var %[1]s in parent workflow file %[3]s", k, path, file)
+		}
+
+		// import all tasks from the import file into the parent file, removing task name prefix
+		re := regexp.MustCompile(`^task\-\d{2}\-?`)
+		for _, tx := range wx.Tasks {
+			tx.Name = re.ReplaceAllString(tx.Name, "")
+			w.Tasks = append(w.Tasks, tx)
```
Comment (neolit123) [inline kinder/pkg/test/workflow/workflow.go]: is it adding the tasks of the imported workflow at the bottom of the already existing tasks of the parent workflow?

Comment (fabriziopandini) [reply kinder/pkg/test/workflow/workflow.go]: It is replacing the import task with tasks in the imported workflow (not adding at the bottom)
 e.g.

workflow A
- Task 1 
- Task 2 import workflow B
- Task 3 

And

workflow B
- Task 1 
- Task 2 

Will became
- Task 1 (from A/task1)
- Task 2 (from B/task1)
- Task 3 (from B/task2)
- Task 4 (from A/task2)

Comment (fabriziopandini) [reply kinder/pkg/test/workflow/workflow.go]: You can think at this as an include

Comment (neolit123) [reply kinder/pkg/test/workflow/workflow.go]: thanks for the example
/lgtm

**Pull request merged**